sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -1,49 +1,62 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-  
+
 import { AppConfigCredential } from "./appConfigCredential";
 import { TokenCredential, URLBuilder } from "@azure/core-http";
 import { AppConfiguration } from "./generated/src/appConfiguration";
+import { PageSettings, PagedAsyncIterableIterator } from "@azure/core-paging";
+import "@azure/core-asynciterator-polyfill";
```

Comment:
I think importing this pollyfill means that this package.json can no longer claim `sideEffects: false`. (Just means we might have to update the package.json)

@bterlson Is my thinking correct here?

Reply:
My understanding is that side effects are tracked on a per-package basis, so bundlers when processing that import will choose to eliminate it or not based on the polyfill's sideEffects flag. Would be good to confirm this somehow, however...

Reply:
We could create a node webpack bundle, import the app config client and test on node.js 8 :-D  That should give us an answer with webpack anyway.

Reply:
@bterlson , @chradek: Am I doing the incorrect thing by requiring the polyfill? I saw at least two ways to do this and I wasn't entirely sure what the right one was.

I assume we want support a browser bundle at some point across all modules - is it related to that?

Reply:
You're doing the right thing. I just wasn't sure if importing this polyfill means this package is now considered as having side-effects, or if bundlers look at the polyfill package instead. From what Brian said it's likely the latter, which means you don't have to change your package.json.